lib/helpers.js

```diff
@@ -232,11 +235,36 @@ function writeSourceMaps(file, destPath, callback) {
   }
 
   // Append source map comment
-  file.contents = appendBuffer(file.contents, comment);
+  file.contents = appendBuffer(file.contents, new Buffer(comment));
 
   callback(null, file, sourceMapFile);
 }
 
+function appendBuffer(buf1, buf2) {
+  var eol;
+  if (buf1.slice(-2).equals(bufCR)) {
+    eol = bufCR;
+  } else if (buf1.slice(-1).equals(bufNL)) {
+    eol = bufNL;
+  } else {
+    eol = bufEOL;
+  }
+  return Buffer.concat([buf1, buf2, eol]);
+}
+
+function normalizePath(fp) {
+  // A win32 path is forbidden to use '/' in a path element.
+  var isWin32Path = !fp.includes('/') && fp.includes('\\');
+  if (isWin32Path) {
+    var parsed = path.win32.parse(fp);
+    fp = fp.replace(/\\/g, '/');
+    if (parsed.root && !fp.startsWith('/')) {
+      fp = '/' + fp;
```

Comment:
Why does this add `/` at the beginning if the parsed has a root? In the `normalize-path` tests, it doesn't add one: https://github.com/jonschlinkert/normalize-path/blob/master/test.js#L49